modules/core/src/shaderlib/project32/project32.js

```diff
@@ -25,6 +25,10 @@ vec4 project_position_to_clipspace(
   vec3 position, vec3 position64Low, vec3 offset, out vec4 commonPosition
 ) {
   vec3 projectedPosition = project_position(position, position64Low);
+  if (project_uProjectionMode == PROJECTION_MODE_GLOBE) {
```

Comment:
Nit: A little comment perhaps?

Reply:
It would probably be good to have a test case for such feature additions. 

Generally it is pretty obvious that the this change is going to make the project module harder to maintain and refactor, and we need to give the next person who is going to try to refactor this a good setup for catching breakages.